nservicebus/sagas/timeouts.md

```diff
@@ -10,15 +10,15 @@ related:
 
 Assumptions can not be made in a message-driven environment regarding the order of received messages and exactly when they will arrive. While the connection-less nature of messaging prevents a system from consuming resources while waiting, there is usually an upper limit to a waiting period that the business dictates. 
 
-The upper wait time is modeled in NServiceBus as a `Timeout`:
+The upper wait time is modelled in NServiceBus as a `Timeout`:
 
 snippet: saga-with-timeout
 
-After calling `RequestTimeout<T>`, a timeout message will be persisted and scheduled to run after a specified delay or at specified time.
+After calling `RequestTimeout<T>`, a timeout message will persist and be scheduled to run after a specified delay or at the specified time.
```

Comment:
It sounds strange to say "a timeout message will persist" because it makes it sound like the message persists itself (although this this stems from our rather context-specific usage of the word "persist").

However, I think this entire sentence can be improved.

```suggestion
`RequestTimeout<T>` sends a timeout message which is delivered after the specified delay or at the specified time.
```